front/components/app/ReachedLimitPopup.tsx

```diff
@@ -87,3 +94,109 @@ export function ReachedLimitPopup({
     </Dialog>
   );
 }
+
+export function ReachedMembersLimitPopup({
```

Comment:
Any reasons not to use the same component as `ReachedMessagesLimitPopup` that takes a limit code as prop and renders the appropriate content?